pkgs/development/beam-modules/default.nix

```diff
@@ -43,7 +43,12 @@ let
       elvis-erlang = callPackage ./elvis-erlang { };
 
       # BEAM-based languages.
-      elixir = elixir_1_16;
+      elixir = elixir_1_17;
```

Comment:
This is the main switch for the elixir in nixpkgs. Switching to the new elixir will require a careful testing of each new packages. Could you not change that line ? 
I cant remember if every elixir used in nixpkgs has a fixed version , but just in case, it would be better to switch the main version later in a separate more thoroughly tested pr.

Reply:
For sure, I've reverted this line